drivers/scale_service.go

```diff
@@ -57,7 +57,7 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient *client
 		return http.StatusBadRequest, fmt.Errorf("Invalid service %v", config.ServiceID)
 	}
 
-	if service.Kind != "service" && service.Kind != "loadBalancerService" {
+	if service.Kind != "scalingGroup" && service.Kind != "loadBalancerService" {
```

Comment:
yes, I think we can have the two conditions included to make it backward compatible